src/org/labkey/test/components/ui/grids/CustomizeGridViewDialog.java

```diff
@@ -266,69 +273,93 @@ public String getSelectedShownInGridColumn()
     }
 
     /**
-     * Click on a column in the 'Shown in Grid' panel. Fields added from the Available Fields panel will be added
-     * underneath the selected column in the Shown in Grid panel.
+     * Click on a field in the list of selected fields. Fields added from the Available Fields panel will be added
+     * underneath the selected field.
      *
-     * @param column The column name to click on.
+     * @param field The field name to click on.
      * @return This dialog.
      */
-    public CustomizeGridViewDialog selectShownInGridColumn(String column)
+    public CustomizeGridViewDialog selectFieldFromSelectedFields(String field)
     {
-        return selectShownInGridColumn(column, 0);
+        return selectFieldFromSelectedFields(field, 0);
     }
 
     /**
-     * Click on a column in the 'Shown in Grid' panel. If multiple columns have the same name the index parameter will
+     * Click on a field in the list of selected fields. If multiple fields have the same name the index parameter will
      * identify which one to click on.
      *
-     * @param column The column to click on.
-     * @param index If multiple columns have the same name this will identify which one to click.
+     * @param field The field to click on.
+     * @param index If multiple fields have the same name this will identify which one to click.
      * @return This dialog.
      */
-    public CustomizeGridViewDialog selectShownInGridColumn(String column, int index)
+    public CustomizeGridViewDialog selectFieldFromSelectedFields(String field, int index)
```

Comment:
I'd use a distinct verb for this. Maybe "highlight" since "select" has other meaning.
```suggestion
    public CustomizeGridViewDialog highlightFieldFromSelectedFields(String field, int index)
```